pkg/get/download.go

```diff
@@ -21,7 +21,8 @@ const (
 	DownloadArkadeDir = iota
 )
 
-func Download(tool *Tool, arch, operatingSystem, version string, downloadMode int, displayProgress bool) (string, string, error) {
+func Download(tool *Tool, arch, operatingSystem, version string, downloadMode int, displayProgress bool, dry_runs ...bool) (string, string, error) {
+	dry_run := dry_runs[0]
```

Comment:
Why is there a slice of dry_runs, where you then only index the first item, and assume that there's always one item there? What if there were no items?